runtime-modules/content/src/lib.rs

```diff
@@ -869,6 +916,8 @@ decl_module! {
             // == MUTATION SAFE ==
             //
 
+            let _ = Balances::<T>::slash(&sender, video_state_bloat_bond);
```

Comment:
Do we not put the bloat bond in some treasury account or something? What did we do for bounty pallet bloat bonds, or for storage?